src/topp/FileFilter.cpp

```diff
@@ -408,7 +409,12 @@ class TOPPFileFilter :
     registerTOPPSubsection_("f_and_c", "Feature & Consensus data options");
     registerStringOption_("f_and_c:charge", "[min]:[max]", ":", "Charge range to extract", false);
     registerStringOption_("f_and_c:size", "[min]:[max]", ":", "Size range to extract", false);
-    registerStringList_("f_and_c:remove_meta", "<name> 'lt|eq|gt' <value>", StringList(), "Expects a 3-tuple (=3 entries in the list), i.e. <name> 'lt|eq|gt' <value>; the first is the name of meta value, followed by the comparison operator (equal, less or greater) and the value to compare to. All comparisons are done after converting the given value to the corresponding data value type of the meta value (for lists, this simply compares length, not content!)!", false);
+    registerStringList_("f_and_c:remove_meta", "<name> 'lt|eq|gt' <value>", StringList(), "WARNING: THIS FLAG IS DEPRECATED. USE all:remove_meta instead.   Expects a 3-tuple (=3 entries in the list), i.e. <name> 'lt|eq|gt' <value>; the first is the name of meta value, followed by the comparison operator (equal, less or greater) and the value to compare to. All comparisons are done after converting the given value to the corresponding data value type of the meta value (for lists, this simply compares length, not content!)!", false);
+
+    addEmptyLine_();
+    registerTOPPSubsection_("all","Options that can be applied to all file types");
```

Comment:
I think we can put this into "all" or just move it out of subsections (like the other general stuff e.g. sort).

Reply:
I've added equivalent functionality to all:remove_meta. I left this in for backwards compatibility (it throws a warning when used). If I am okay to remove it I can do so.

Reply:
I meant not to use "all" at all ;)

Reply:
better now?